src/en/hiveworks/src/eu/kanade/tachiyomi/extension/en/hiveworks/Hiveworks.kt

```diff
@@ -178,8 +178,27 @@ class Hiveworks : ParsedHttpSource() {
     // Included to call custom error codes
     override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
         return if (manga.status != SManga.LICENSED) {
+            val uri = Uri.parse(manga.url).buildUpon()
             client.newCall(chapterListRequest(manga))
-                .asObservableSuccess()
+                .asObservable()
+                .doOnNext { response ->
+                    if (!response.isSuccessful) {
+                        if ("smbc-comics" in uri.toString() && response.code == 500) {
+                            // As of March 2025, SMBC chapter list page returns
+                            // status code 500 even though it still has correct data.
+                            // Do not throw an error in this case.
+                            //
+                            // I reported this error to SMBC on 2025-05-28, but even if
```

Comment:
Should wait a week and see if they fix it.

Reply:
Waited and have not heard back nor has the issue been fixed, so I have updated the implementation here with your comments and have re-tested.